DATA/production/calib/hmp-pedestals-processing.sh

```diff
@@ -72,7 +72,7 @@ then
   WORKFLOW+="--use-files "
 fi
 
-WORKFLOW+="--files-basepath 'HMP/Calib/Pedestals' "
+WORKFLOW+="--files-basepath 'HMP' "
```

Comment:
Generally, I like to have `Detector/[Calib, Config]/Path3` but if @shahor02 also agrees, it is fine with me.